src/main/java/ava/task/Task.java

```diff
@@ -0,0 +1,95 @@
+package ava.task;
+
+import ava.Ui;
+import ava.processor.Storage;
+import ava.processor.TaskList;
+
+/**
+ * Abstract class to represent the tasks.
+ */
+public abstract class Task {
+    public Boolean isBye;
```

Comment:
This class variable might be better as `protected`, if access is required - consider writing an accessor?

Reply:
I changed it, but haven't commited. Thanks !